### PR TITLE
Copy edit MONAI Deploy Informatics Gateway docs

### PR DESCRIPTION
**Pull request by Coco-Ben**
### Types of changes

- [ ] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] All tests passed locally by running `./src/run-tests-in-docker.sh`.
- [ ] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.
- [x] User guide updated.
- [ ] I have updated the changelog

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ After inference completes, MIG receives notifications for exporting the results
 
 ## Services
 
-*MONAI Deploy Informatics Gateway* contains the following standard protocols to communicate with your medical devices:
+MIG contains the following standard protocols to communicate with medical devices:
 
 * **DICOM SCP**: C-ECHO, C-STORE
 * **DICOM SCU**: C-STORE
```
Comment (Coco-Ben) [inline docs/index.md:24]: Are the  DICOMweb client and FHIR client protocols subsets of the ACR DSI API? The indenting would suggest so.

File: docs/setup/schema.md [hunk 2]
```diff
@@ -150,11 +150,11 @@ Informatics Gateway validates all configuration options at startup. Any provided
 Informatics Gateway, by default, is configured to writes all logs to the console as well as text files. The behaviors may be changed in the `Logging` section of the `appsettings.json` file.
 
 > [!Note]
-> If the Informatics Gateway is running inside a Docker container, additional configuration may be required to limit the size to prevent filling up storage space. Please refer to [Docker](https://docs.docker.com/config/containers/logging/configure/) for additional information.
+> If the Informatics Gateway is running inside a Docker container, additional configuration may be required to limit the size to prevent filling up storage space. Refer to the [Docker documentation](https://docs.docker.com/config/containers/logging/configure/) for additional information.
 
 #### Log Levels
 
-Log level may be adjusted on a per-module basis. For example, given the following log entries:
+The log level may be adjusted on a per-module basis. For example, given the following log entries:
```
Comment (Coco-Ben) [inline docs/setup/schema.md:157]: This language suggests that the paragraph following the example (on line 166) will be related to it. However, that paragraph doesn't seem to discuss the example.

File: docs/setup/setup.md [hunk 3]
```diff
@@ -58,58 +58,57 @@ docker pull ghcr.io/project-monai/monai-deploy-informatics-gateway:0.1.0
...
+By default, the temporary storage location is set to `/payloads` in the `appsettings.json` file.
 
-To change the temporary storage location, please locate `./InformaticsGateway/storage/temporary` property in the `appsettings.json` file.
+To change the temporary storage location, locate the `./InformaticsGateway/storage/temporary` property in the `appsettings.json` file and modify it.
 
 > [!Note]
-> Calculate the required temporary storage based on the number of studies and the size of each study.
-> Please also consider the AE Title timeout if the AE Title needs to wait a long time before assembling & uploading
+> You will need to calculate the required temporary storage based on the number of studies and the size of each study.
+> Also, consider changing the AE Title timeout if the AE Title needs to wait a long time before assembling and uploading
```
Comment (Coco-Ben) [inline docs/setup/setup.md:99]: It's unclear in the language whether the reader needs to consider changing the value of the AE Title timeout or 
or enabling it altogether (i.e. is it enabled by default?). I assumed the former when rewriting.

File: docs/index.md (hunk 1 above)
Comment (mocsharp) [reply docs/index.md:24]: They are not, but the only way of using them is by calling the ACR API.

File: docs/setup/schema.md (hunk 2 above)
Comment (mocsharp) [reply docs/setup/schema.md:157]: How about something like:

By default, the `Monai` namespace is to log all `Information` level logs. However, log level may be adjusted on a per-module basis. For example, given the following log entries:

```
 14:26:13 info: Monai.Deploy.InformaticsGateway.Services.Connectors.WorkloadManagerNotificationService[0]
      MONAI Workload Manager Notification Hosted Service is running.
 14:26:13 info: Monai.Deploy.InformaticsGateway.Services.Storage.SpaceReclaimerService[0]
      Disk Space Reclaimer Hosted Service is running.
```

If additional information is required to debug the **WorkloadManagerNotificationService** module or to turn down the noise, add a new entry under the `LogLevel` section of the configuration file to adjust it:

```
 "Logging": {
    "LogLevel": {
      "Monai": "Information",
      "Monai.Deploy.InformaticsGateway.Services.Connectors.WorkloadManagerNotificationService": "Debug",
      ...
```

File: docs/setup/setup.md (hunk 3 above)
Comment (mocsharp) [reply docs/setup/setup.md:99]: Users shall consider changing the default timeout of 5 seconds depending on how they are sending the DICOM dataset.

Comment (Coco-Ben) [reply docs/setup/setup.md:99]: Got it. No changes required then.

**Review by mocsharp — APPROVED** [review]

**Pull request merged**